Documentation/PLUGIN.md

```diff
@@ -123,6 +123,40 @@ The possible values for `Visibility` are:
 * `Public`: The visibility on the types is set to `public` so the types will
   be exposed outside the module they are compiled into.
 
+
+##### Generation Option: `ProtoPathModuleMappings` - Swift Module names for proto paths
+
+By default, the sources generated assume all the generated types will be in the same
+module. However, since protos can reference types from another proto file, there is
+the possibility that those generated files might need to end up different modules.
+This options allow you to provide a mapping of module names the and proto files that
```

Comment:
This option allows you to specify that the code generated from the proto files will be distributed in multiple modules.